### ISSUE TITLE
Support for more than one xx chain asset 

### ISSUE THREAD

**Issue by armchairancap** [armchairancap/polkadot-js-tokengated-website#2]
Currently this PoC code supports single asset with one setting and protected pages do not show its name which is fine, because it's unambiguous which asset is checked as long as the site owner makes that clear.

But it'd be nice to support multiple pages and have the ability to avoid modifying app code or rebuilding container.

For example:

- Move `assetId` to `.env`
- Provide support for multiple asset IDs and names
  - Specifying asset IDs _and_ names frees us from adding code to look them up dynamically from asset metadata

```raw
assets: [
    {
        assetID: '5',
        assetName: 'JUNK',
        level: [10,20,50,100]
    },
    {
        assetID: '6',
        assetName: 'MEMEMBERSHIP2024'
        level: [0,1,2]
    } 
]
````
- Levels - could be optionally provided for multiple levels, e.g. with 4 levels some perk could be `$(LEVEL).COUNT*$AMOUNT` and in other cases it could be simple Free, Standard, Premium (0,1,2).

Pull requests are welcome.

**Comment by armchairancap** [comment]
Similar configuration can be added for XX coin, but that's relatively easy compared to handling assets.

**Comment by armchairancap** [comment]
Kind of solved in v1.3.0 as it basically works - config file is there and can be matched against - but actual logic needs to be customized. 

Can be revisited based on specific use case or requirements in the future.
